packages/sdk/src/BoostCore.ts

```diff
@@ -441,9 +445,24 @@ export class BoostCore extends Deployable<
       }
     }
 
+    // If not providing a custom validator, use either Boost's mainnet or testnet EOA, depending on provided chain id and given chain configurations
     if (!payload.validator) {
+      const chains = getChains(options.config).filter(
+        (chain) => !!this.addresses[chain.id] && chain.id === chainId,
+      );
+      const chain = chains.at(0);
+      if (!chain)
+        throw new InvalidProtocolChainIdError(
+          chainId,
+          Object.keys(this.addresses),
+        );
+      const testnet = chain.testnet || chain.id === 31337;
       payload.validator = this.SignerValidator({
-        signers: [payload.owner],
+        signers: [
+          (testnet
+            ? BoostValidatorEOA.TESTNET
+            : BoostValidatorEOA.MAINNET) as unknown as Address,
+        ],
         validatorCaller: coreAddress,
       });
```

Comment:
if we are going to reuse the same signers, is it possible if we can pass in existing SignerValidator contracts that have been deployed? I can see possible gas savings

Or is there some consideration for having to create new SignerValidators for each boost?

Reply:
it could definitely be worth referencing one existing validator, but idk how to do that. @topocount has machinations to do it for openallowlist and now we'll throw signervalidator on the pile